beaver/worker/worker.py

```diff
@@ -53,7 +53,8 @@ def __init__(self, beaver_config, queue_consumer_function, callback, logger=None
         self._folder = self._beaver_config.get('path')
         self._last_file_mapping_update = {}
         self._logger = logger
-        self._proc = None
+        self._number_of_consumer_processes = int(self._beaver_config.get('number_of_consumer_processes'))
```

Comment:
@josegonzalez 
Thanks, missed it.
I added it to worker/tail_manager.py but I don't see anything related in worker/tail.py. Can you focus on where should I add the change in worker/tail.py?